### DISCUSSION TITLE
How to check my SD-WebUI's version is 0.3.0 or 0.10.0？Does the words in my CMD  “diffusers version 0.14.0 ” have any relation to those mentioned previously?

### DISCUSSION THREAD

**Discussion by glorydanking** [d8ahazard/sd_dreambooth_extension#1135] [Q&A]
As the Installation described in [Dreambooth Extension for Stable-Diffusion](https://github.com/d8ahazard/sd_dreambooth_extension) . 
How to check my SD-WebUI version ?

"We also need a newer version of diffusers, as **SD-WebUI** uses **version 0.3.0**, while DB training requires >= **0.10.0**. Not having the right diffusers version is the cause of the 'UNet2DConditionModel' object has no attribute ' enable_gradient_checkpointing' error message, as well as safety checker warnings."

newer version of diffusers means "diffusers version 0.14.0" needs upgrade ?
 if so, how to upgrade. 
 if not, how should i understand 
"a newer version of diffusers" & "SD-WebUI version" & change my SD-WebUI version "0.3.0" or 0.10.0, then check them？

my SD-WebUI cmd as followed
```
Python 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Commit hash: <none>
Installing requirements for Web UI

If submitting an issue on github, please provide the full startup log for debugging purposes.

Initializing Dreambooth
Dreambooth revision: 3324b6ab7fa661cf7d6b5ef186227dc5e8ad1878
Successfully installed accelerate-0.17.1 fastapi-0.90.1 gitpython-3.1.31 requests-2.28.2 starlette-0.23.1 transformers-4.26.1

Does your project take forever to startup?
Repetitive dependency installation may be the reason.
Automatic1111's base project sets strict requirements on outdated dependencies.
If an extension is using a newer version, the dependency is uninstalled and reinstalled twice every startup.

[+] xformers version 0.0.17rc482 installed.
[+] torch version 2.0.0+cu118 installed.
[+] torchvision version 0.15.1+cu118 installed.
[+] accelerate version 0.17.1 installed.
[+] diffusers version 0.14.0 installed.
[+] transformers version 4.26.1 installed.
[+] bitsandbytes version 0.35.4 installed.

```
“diffusers version 0.14.0 installed”，does this diffusers （0.14.0) and SD-WebUI（version 0.3.0）matters？

**Comment by wb1357076878** [comment]
my SD-WebUI cmd as followed:
Does your project take forever to startup?
Repetitive dependency installation may be the reason.
Automatic1111's base project sets strict requirements on outdated dependencies.
If an extension is using a newer version, the dependency is uninstalled and reinstalled twice every startup.

[!] xformers NOT installed.
[!] torch version 1.12.1 installed.
[!] torchvision version 0.13.1 installed.
[+] accelerate version 0.17.1 installed.
[+] diffusers version 0.14.0 installed.
[+] transformers version 4.26.1 installed.
[+] bitsandbytes version 0.35.4 installed.

#######################################################################################################
#                                       LIBRARY ISSUE DETECTED                                        #
#######################################################################################################
#
# torch is below the required 1.13.1+cu116 version.
# torchvision is below the required 0.14.1+cu116 version.
#
# Dreambooth may not work properly.
#
# TROUBLESHOOTING
# 1. Fully restart your project (not just the webpage)
# 2. Update your A1111 project and extensions
# 3. Dreambooth requirements should have installed automatically, but you can manually install them
#    by running the following 4 commands from the A1111 project root:
cd venv/Scripts
activate
cd ../..
pip install -r ./extensions/sd_dreambooth_extension/requirements.txt

How to install xformers torch torchvision,cause my webui cant launch again,when i pull the latest sd-webui version

**Comment by glorydanking** [comment]
感谢你的来信，我会尽快回复给您，祝生活愉快，工作顺利~
